vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -180,14 +217,25 @@ def main(args):
             )
         )
     else:
-        logger.info("Constructing a test table")
+        logger.info("Constructing a test Table")
         ht_final = ht_original.annotate(
             VRS_Allele=ht_annotated[ht_original.locus, ht_original.alleles].VRS_Allele
         )
 
-    logger.info(f"Outputting final table at: {output_paths_dict[args.version]}")
-    ht_final.write(output_paths_dict[args.version],overwrite=args.overwrite)
-    # hl.write_table(ht_final,output_paths_dict[args.version])
+        logger.info(f"Outputting final table at: {output_paths_dict[args.version]}")
+        ht_final.write(output_paths_dict[args.version], overwrite=args.overwrite)
```

Comment:
was this temporarily indented?

Reply:
I'm not sure what these means? I used Black for formatting, does the indentation still look off ? I'd be happy to change indentation if something looks off

Reply:
the way it's written right now, the final output if using "3.1.2" as version will not get output

Reply:
Oh okay I see it now, will fix!